--- zziplib-0.13.58/docs/Makefile.am~	2009-08-23 13:38:21.000000000 +0200
+++ zziplib-0.13.58/docs/Makefile.am	2009-08-25 10:04:05.000000000 +0200
@@ -180,10 +180,10 @@
 	; fi
 
 install-man3 : manpages.tar
-	$(mkinstalldirs) $(DESTDIR)$(mandir)/man3
+	$(mkinstalldirs) $(DESTDIR)$(mandir)
 	P=`pwd` ; test -s manpages.tar || exit 1 \
-	; cd $(DESTDIR)$(mandir) && $(PAX_TAR_EXTRACT) $$P/manpages.tar \
-        ; true
+        ; cd $(DESTDIR)$(mandir) && $(PAX_TAR_EXTRACT) $$P/manpages.tar \
+	; true
 
 unpack : manpages.tar htmpages.tar
 	test -s manpages.tar && test -s htmpages.tar
