--- zziplib-0.13.47/m4/ax_enable_builddir_uname.m4.orig	2006-08-07 22:58:48.000000000 +0200
+++ zziplib-0.13.47/m4/ax_enable_builddir_uname.m4	2006-08-24 15:06:18.215971000 +0200
@@ -96,7 +96,7 @@
 AS_VAR_PUSHDEF([SED],[ax_enable_builddir_sed])dnl
 pushdef([END],[Makefile.mk])dnl
   SRC="$ax_enable_builddir_srcdir"
-  if test ".$SUB" = "." ; then
+  if test ".$SUB" = ".." ; then
     if test -f "$TOP/Makefile" ; then
       AC_MSG_NOTICE([skipping TOP/Makefile - left untouched])
     else
